changes/pr3299.yaml

```diff
@@ -0,0 +1,2 @@
+enhancement:
+  - "Allow copying of directories into Docker image - [#3299](https://github.com/PrefectHQ/prefect/pull/3299)"
```

Comment:
You should add yourself as a contributor here!  To do so, include:

```
contributor:
  - "[Contributor Name](https://github.com/sevberg)"
```
at the bottom of this file